stock_operating_unit/model/stock_move.py

```diff
@@ -9,35 +10,47 @@ class StockMove(models.Model):
     _inherit = "stock.move"
 
     operating_unit_id = fields.Many2one(
-        'operating.unit',
-        compute="_compute_operating_unit_id", store=True,
-        string='Source Location Operating Unit',
+        "operating.unit",
+        compute="_compute_operating_unit_id",
+        store=True,
+        string="Source Location Operating Unit",
     )
 
     operating_unit_dest_id = fields.Many2one(
-        'operating.unit',
+        "operating.unit",
         compute="_compute_operating_unit_dest_id",
-        string='Dest. Location Operating Unit',
+        store=True,
+        string="Dest. Location Operating Unit",
     )
 
-    @api.depends('location_id', 'location_id.operating_unit_id',
-                 'picking_type_id', 'picking_type_id.warehouse_id',
-                 'picking_type_id.warehouse_id.operating_unit_id')
+    @api.depends(
+        "location_id",
+        "location_id.operating_unit_id",
+        "picking_type_id",
+        "picking_type_id.warehouse_id",
+        "picking_type_id.warehouse_id.operating_unit_id",
+    )
     def _compute_operating_unit_id(self):
         for item in self:
             # For dropshipping
-            item.operating_unit_id = \
-                item.location_id.operating_unit_id or \
-                item.picking_type_id.warehouse_id.operating_unit_id
+            item.operating_unit_id = (
+                item.location_id.operating_unit_id
+                or item.picking_type_id.warehouse_id.operating_unit_id
+            )
 
-    @api.depends('location_id', 'location_id.operating_unit_id',
-                 'picking_type_id', 'picking_type_id.warehouse_id',
-                 'picking_type_id.warehouse_id.operating_unit_id')
+    @api.depends(
+        "location_id",
+        "location_id.operating_unit_id",
```

Comment:
Should this not be location_dest_id.operating_unit_id
Because in the method you are using the location_dest_id field and not the location_id field.

Reply:
Now it looks good to me!